language/diem-framework/releases/artifacts/current/transaction_script_builder.rs

```diff
@@ -3549,9 +3549,9 @@ pub fn encode_add_currency_to_account_script_function(currency: TypeTag) -> Tran
     TransactionPayload::ScriptFunction(ScriptFunction::new(
         ModuleId::new(
             AccountAddress::new([0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1]),
-            Identifier::new("AccountAdministrationScripts").unwrap(),
+            ident_str!("AccountAdministrationScripts").to_owned(),
```

Comment:
Maybe we need to change the ModuleId::new function to take in `Into<Identifier>` so that we don't need so many of these `to_owned()` calls. Not super important though

Reply:
I think I'll probably leave that out for this PR